NewHorizons/Components/Props/NHSupernovaPlanetEffectController.cs

```diff
@@ -184,7 +184,7 @@ public void Update()
                 {
                     float collapseProgress = SunController.GetCollapseProgress();
 
-                    if (_ambientLight != null)
+                    if (_ambientLight != null && _ambientLightOrigIntensity != null)
                     {
                         for (int i = 0; i < _ambientLight.Length; i++)
                         {
```

Comment:
do we need to null check each ambientlight here too like we do in the for loop below? what about _ambientLightOrigIntensity? idk just asking

Reply:
u do it!

Reply:
idk doing this fixed it so idc